redirects.yaml

```diff
@@ -265,7 +212,7 @@ cloudbox/private/?: "/openstack"
 cloudbox/support/?: "/support"
 cloudbox/training/?: "/cloud/training"
 cluster/?: "/cloud/training"
-communi[^/]*/?: "https://community.ubuntu.com/"
+communi*/?: "https://community.ubuntu.com/"
```

Comment:
http://0.0.0.0:8001/community does not redirect.

Reply:
I think I am going to send all community/.* links to community.u.c

Reply:
We had a issue a few week back which asked to have a community redirects deep link to the new community site. I did the work so we should have them all linking directly now.